_config.yml

```diff
@@ -36,6 +36,12 @@ twitter:
 # Build settings
 markdown: kramdown
 theme: minima
+kramdown:
+  parse_block_html: false
+  # This is the default, but be explicit as some EIPs depend on it
+  auto_ids: true
+  # This is to ensure more determistic behaviour
+  auto_id_stripping: true
```

Comment:
Link generation is explained here: https://kramdown.gettalong.org/converter/html.html#auto-ids

Newer version of kramdown changed the rules, hence the changes to existing EIPs.